### PR TITLE
Add 'Was this Helpful?' section to footer

### PR DESCRIPTION
**Pull request by patmellon**
This PR adds the CSS and HTML for the "Was this Helpful?" section that will appear in the tutorials footer.

### CODE REVIEW

File: pytorch_sphinx_theme/footer.html [hunk 1]
```diff
@@ -10,7 +10,22 @@
     </div>
   {% endif %}
 
-  <hr/>
+  {% if theme_pytorch_project == 'tutorials' %}
+
+    <hr class="helpful-hr hr-top">
+      <div class="helpful-container">
+        <div class="helpful-question">Was this helpful?</div>
+        <div class="helpful-question yes-link" data-behavior="was-this-helpful-event" data-response="yes">Yes</div>
```
Comment (JoelMarcey) [inline pytorch_sphinx_theme/footer.html:18]: I assume `was-this-helpful-event` is defined somewhere already.

**Review by JoelMarcey — APPROVED** [review]
Small comment ensuring we have a definition.

File: pytorch_sphinx_theme/footer.html (hunk 1 above)
Comment (patmellon) [reply pytorch_sphinx_theme/footer.html:18]: `was-this-helpful-event` is used in this [tutorials PR](https://github.com/pytorch/tutorials/pull/443) to track the event via FB pixels and Google Analytics.

Comment (JoelMarcey) [reply pytorch_sphinx_theme/footer.html:18]: Oh right - we just hadn't merged that yet 😄

**Comment by JoelMarcey** [comment]
Don't merge this until we merge https://github.com/pytorch/tutorials/pull/443

**Comment by JoelMarcey** [comment]
cc @brucejlin ^

**Pull request merged**